var/spack/repos/builtin/packages/hdf5-vol-cache/package.py

```diff
@@ -17,7 +17,7 @@ class Hdf5VolCache(CMakePackage):
     version("v1.1", tag="v1.1")
     version("v1.0", tag="v1.0")
 
-    depends_on("hdf5@1.13: +mpi +threadsafe")
+    depends_on("hdf5@1.14: +mpi +threadsafe")
```

Comment:
Possibly unrelated, but I ran into this error when building with OneAPI. Are all of the versions here working with 1.14?

https://gitlab.spack.io/spack/spack/-/jobs/7089562

Reply:
@kwryankrattiger , thank you for the error report.
It's interesting that the oneapi@2021.4.0 works: https://my.cdash.org/build/2344393 

@lrknox , can you test with the latest oneapi and report it to CDash?